cmd/snap/cmd_find.go

```diff
@@ -84,8 +84,22 @@ func getPriceString(prices map[string]float64, suggestedCurrency, status string)
 	return formatPrice(price, currency)
 }
 
+type SectionName string
+
+func (s *SectionName) Complete(match string) []flags.Completion {
+	// TODO cache the result
```

Comment:
Should be cached on the snapd daemon end rather than here, which means we can cache in memory instead of on disk. So it's okay to drop the TODO here.